src/cli/utils.js

```diff
@@ -78,7 +78,7 @@ async function lstatSafe(filePath) {
     return await fs.lstat(filePath);
   } catch (/** @type {any} */ error) {
     /* c8 ignore next 3 */
-    if (error.code !== "ENOENT") {
+    if (error.code !== "ENOENT" && error.code !== "ENAMETOOLONG") {
```

Comment:
I wonder how it will work if it's not a glob pattern. Maybe we should not ignore it?

[`isDynamicPattern`](https://github.com/mrmlnc/fast-glob#isdynamicpatternpattern-options) can be useful.